### PR TITLE
[ci skip]add NodeGrid bind

### PR DESCRIPTION
**Pull request by jianglong0156**
add NodeGrid bind
testcase: js-test->effectTests.

### CODE REVIEW

**Comment by jianglong0156** [comment]
@pandamicro  Can you help to review it

**Comment by jianglong0156** [comment]
the testcase relate pr:https://github.com/cocos2d/cocos2d-x/pull/13357

File: cocos/scripting/js-bindings/script/jsb_cocos2d.js [hunk 1]
```diff
@@ -2693,6 +2693,30 @@ _p.unscheduleUpdateForTarget = _p.unscheduleUpdate;
 _p.unscheduleAllCallbacksForTarget = _p.unscheduleAllForTarget;
 
 
+cc._NodeGrid = cc.NodeGrid;
+cc.NodeGrid = function(rect){
+    if (!(this instanceof cc.NodeGrid)){
+        cc.error("NodeGrid Constructor can not called as a function, Please use new");
```
Comment (pandamicro) [inline cocos/scripting/js-bindings/script/jsb_cocos2d.js]: NodeGrid's constructor can not be called as a function, please use 'new cc.NodeGrid()'

**Comment by pandamicro** [comment]
Review done, please modify the error message, no other problem

**Comment by jianglong0156** [comment]
improve the error message

**Pull request merged**